Manifests/ManagedPreferencesApplications/com.vpntracker.365mac-config.plist

```diff
@@ -0,0 +1,344 @@
+	4wWdzDnAsmw5bf5N5wbTOQDh9QBClMHN3dqC570NM9a8HIWByxDhNucXNQqweJl8Abw4
+	iG9//js4fvi4Jp803TypkYhRxDViTrneQ8DAtgPHDYwAqkJeAYhudQpDmSySM2887yZr
+	PClEmMlTEyAE84r2jGUbz2V5iknmY+SNDvMKeqt7DWasewXyA5cMeckqNy8h+UTBnP6+
+	9flv4+TRkybty6STGmFTC8iGI+BYxIwheYsCNCRvIAGzxuA4TZTAQE6ideltpyRnVpRD
+	1QJEs0CnEm1onZ1Jdd1yanAgY5Qfdp8zULcWLYvvDJxnJciI87bIYlqiCD83gG987lv6
+	QHMKdXUJQ15Dk9eEHVSI6tdRpy0FK3Q8IG4AEv3mSBoQKOSyWaiGxZebuhb3cL/0zAQY
+	8aVi2VVr7q2vONCbs4Ss3BSZujeHHQm/mAX38sZ1Q54VNfmcJl9CKduPr3/mmzhz/HSE
+	PDHkI+mtoQxp+nuEA3LRdMcwMRlASON+d18RM9e/bC+hVIVr1+NUBTAxPAnzSlZT5+Le
+	uhuee6SvZ8CkYFD/CtlMGgWd+kRo8l4O0s8HzidLKA714Wuf/jrOnjyD+qjzToU4Des6
+	7OoIiBvCIWTEyQjxSh8CUcikM0DrunMzltx0oVyyUfc11JR7QDQLuF+0F9/7l7vSqjXr
+	FfJQMhCgWCji9I4HUe8U0KRrvVkTb26UyKd7DPlzp88Z5+MR8lHnLSta24HjClGnA9Ko
+	kA6Jy/A183BtyGXz7/mbrUpJoq/9h3wrLEamT3lbSdZPL8x/7pueOPmz975gdrsFKQmo
+	HcOBx38L/XcDtM6ehVRNCr7PsPWRp3H1ytWK88TAHa53pUEqHR2gCM/0hn64uQEK4/61
+	mVLg0tUMWje+aUvjzHn9rJiLhW+NCCABqKoEyOfzTClFRvxxVPmlvDN7xa1nBy7+xY7L
+	27+5vqWlHkDZOYKTh/Zj347d4AIQUsHVbOtr44i7ZfKVTm9FU14Npz0hEZIKgfsYN0z2
+	9HSn4cx74ZF5N7/4kCbvRtO98pIUi0UGQFYjgOzt7c2PmkL6NSvm3cV3/+nu3eme1LWj
+	v1za0FBr9u1UMg5qCTBuFm8cjjlUC1DZ3yMpb1ENAgS8w88gIJMgDgRz9PVk4DVvPLv6
+	Ra99UnDPHu3IS3T09/cXAPBqBFDnzp0bZIwpHWRkE1FSEqm4vfyFr3lqL+Ok5+ivl9TV
+	JyElNa7yiACOHSJKPnCeEI0w3fUwOeqBeL3dGbC2286ufNW7N2mOSnBGAYwqwPnz5wcB
+	iKpKQD8ec3VgYMBLJpNURu6Bw+KSQlAllbX8Ba9+6nCspti7+1/XNNZQ2EkHjAf35uEx
+	1oo6H7mDC8lHnxgbTwJKCZRguNxdRHzRSw+tfOGrn6Zl8synIJF5IuQ9z5P6KdIrYwlg
+	YfQgugdAP2e3RD9dUe/7vqzwHnlOIFJw2tS1/DJSMwevnjnWYrFBN5V0Kg0uJE8CUAyf
+	5UkZZHIPLJOKYLlsDj35Gq/19tc9ueieP9+lbadSY5yHJi3tfl4/6reZcz4AQE1aAA2R
+	zWYbXvSiFy1kjMnhXqARlkFQHloEzuz61s6Bhjnrz6eHPDtz7dx0ByVSvsFxbBqSDjOi
+	QigkpwCMDtMcIVEqFNCblpDtt51a9JL3bG5bvP4C9wpueQ3jbePlB7y+/e1vH9y1a9dW
+	AF4VApggFy5cKOqnwecvXry4vlQqCUSjTF4jtEg7YbmJJGtZuO58Ysbyy5kct3L93XXK
+	z1HHluEdnhFh2HkSVTy8pqGkubPM54oYyNtSNG84P/vu1z8x7/Y/3OsmajzhF5wI0VHJ
+	69K1jx8/nv/whz98nzbwMgBe7ePytkayra1tww9+8IOXz5o1yy0UChyRkEIQ0xAr5aD0
+	//U1EMsRZRrZ3ksNvad2duXO7+6QQ+enOyKrk7IsBgnFCBdstk0uFJig4DQlSG3HQG3n
+	mvPNi2890zhzfh8hgGCeHSU8Vuq7rks1aaofjNqknxbdBGBIg1UrANGIadTrLLjti1/8
+	4t0dHR2uLgsW/VAlKycvgES2TQI9UtsRlu0KzplVHOyuyfVcmFbov9joDV2rE8VMInxY
+	mtgut+O1JbeudSjZ1DFY09rVn5rWPuTEE0xxZpXvQ0YhOpbzljYK733ve7c//vjjDwLo
+	raS/rFYApFIpSzfDGIDGmTNnrn//+9+/8fbbb58mdJS7qw4V9oWRh6YwSKXQqeVI6jiS
+	EEsGwgkqlSRGOUqUvq5RnkKSSnOz9OtJ/tDCPBBJtPO27vi5j33sYzsPHDjwFIAejVKY
+	/lULEHZSnU5xAHUa85773Oeu0g9Pz126dGmNDitYtIKGigowmVCoxLhpPXqQIMKTnjx9
+	+nTxN7/5zcVf/vKXe3W/OgYgTQgpTfTjiXCSiZS2KuWQ0GjQmDFnzpzZujm2dHZ21k6f
+	Pj1WFqPsAABiMDGRakmHC5baEKEzU+hziq9/y5DTza5PPxN8SUp5CcCARjF0/j/yN0NU
+	w9FwNWIRQeKVa44GxX9NyAo5v0I0hFe5xjTEf9aPpqwIbA1aAZn6nFWFioyyAlEBr/x/
+	0ilHMPUgIf4LyUdDjYLrcT2uR3Xx71fhewWLJrR8AAAAAElFTkSuQmCC
+	</data>
+	<key>pfm_last_modified</key>
+	<date>2022-11-11T09:00:00Z</date>
+	<key>pfm_platforms</key>
+	<array>
+		<string>macOS</string>
+	</array>
+	<key>pfm_subdomain</key>
+	<string>Onboarding</string>
```

Comment:
Is this intentional?  https://github.com/ProfileCreator/ProfileManifests/wiki/Subdomains

Reply:
It was intentional as I was planning ahead: we might want to add additional MDM support and I thought it would be a good idea to have separate manifests for onboarding (user management) and other, unrelated settings. Should we remove it?

Reply:
It's more about user experience.  The reason we had to implement this feature is that there are a number of profile payloads like Energy Saver (https://developer.apple.com/documentation/devicemanagement/energysaver) and Time (https://developer.apple.com/documentation/devicemanagement/timeserver) - 2 completely different and unrelated things - but which Apple ultimately has handled in the same payload: `com.apple.MCX`.

All that the `pfm_subdomain` key does this allows us to do is treat these (and related) items as _separate_ manifests & payloads in the interface but which ultimately export to the same `com.apple.MCX`.

So, if you want admins to do that in this case that's one thing, but given how you've named the plist - `com.vpntracker.365mac-config.plist` - and that you've just specified 'Onboarding', I'm not sure what your intent is there.  Ultimately, if this is really all going to center around the same core thing (VPN Tracker) then you probably just want to have it all within 1 manifest and separate out the settings into separate menus (Zoom and a number of other payloads do this to group settings together to make things easier to find and not just be a long list of options).